doc/mayastor-client.md

```diff
@@ -280,9 +280,8 @@ We are maxing out at roughly 90MB, as this a 1 GbE network that is to be expecte
 Now, let's disconnect it and create a Nexus that that consumes one of the NVMe targets and rerun the test:
 
 ```bash
-    nvme disconnect -d {/dev/nvme1,/dev/nvme2}
-    mayastor-client nexus create `uuidgen -r` 64MiB 'nvmf://192.168.1.2/nqn.2019-05.io.openebs:cnode2
-        nvmf://192.168.1.2/nqn.2019-05.io.openebs:cnode1'
+nvme disconnect -d {/dev/nvme1,/dev/nvme2}
+mayastor-client nexus create `uuidgen -r` 64MiB 'nvmf://192.168.1.2/nqn.2019-05.io.openebs:cnode2 nvmf://192.168.1.2/nqn.2019-05.io.openebs:cnode1'
```

Comment:
Same change as above

Reply:
Again, not clear if this has been changed. Also better if at all possible to try and prevent line breaks in the middle of an argument